src/pudl/transform/eia860.py

```diff
@@ -593,24 +575,18 @@ def _core_eia860__boiler_generator_assn(
     * Drop duplicate rows.
 
     Args:
-        raw_eia860__boiler_generator_assn (df): Each entry in this dictionary of DataFrame objects
-            corresponds to a page from the EIA860 form, as reported in the Excel
+        raw_eia860__boiler_generator_assn: Each entry in this dictionary of DataFrame
+            objects corresponds to a page from the EIA860 form, as reported in the Excel
             spreadsheets they distribute.
 
     Returns:
         Cleaned ``_core_eia860__boiler_generator_assn`` dataframe ready for harvesting.
     """
     # Populating the 'core_eia860__scd_generators' table
-    b_g_df = raw_eia860__boiler_generator_assn
-
-    b_g_df = pudl.helpers.convert_to_date(b_g_df)
-    b_g_df = pudl.helpers.convert_cols_dtypes(df=b_g_df, data_source="eia")
-    b_g_df = b_g_df.drop_duplicates()
-
     b_g_df = (
-        pudl.metadata.classes.Package.from_resource_ids()
-        .get_resource("core_eia860__assn_boiler_generator")
-        .encode(b_g_df)
+        pudl.helpers.convert_to_date(raw_eia860__boiler_generator_assn)
+        .pipe(pudl.helpers.convert_cols_dtypes, data_source="eia")
+        .drop_duplicates()
```

Comment:
Removed the encoding step because it's the last part of the transform, and now happens in the harvesting process for this table.

Reply:
hm well but is it at all helpful for the eia bga/unit_id_pudl assignment?

Reply:
Ah, maybe this is why I had to add an explicit encoding to the BGA function?

Reply:
yes. okay i added a comment over in the transfrom.eia module about this so i'm resolving this thread